build.sbt

```diff
@@ -880,8 +923,11 @@ lazy val `project-manager` = (project in file("lib/scala/project-manager"))
   .dependsOn(`polyglot-api`)
   .dependsOn(`runtime-version-manager`)
   .dependsOn(`library-manager`)
+  .dependsOn(`logging-utils-akka`)
+  .dependsOn(`logging-service`)
   .dependsOn(pkg)
   .dependsOn(`json-rpc-server`)
+  .dependsOn(`logging-logback` % Runtime)
```

Comment:
Does it mean that the `logging-service` is an interface and the `logging-logback` is the implementation of the interface? I'd call it `logging-service-logback` then to be more clear